state/sqlite/sqlite_dbaccess.go

```diff
@@ -250,25 +250,32 @@ func (a *sqliteDBAccess) Get(parentCtx context.Context, req *state.GetRequest) (
 	}
 
 	// Concatenation is required for table name because sql.DB does not substitute parameters for table names
-	stmt := `SELECT key, value, is_binary, etag FROM ` + a.metadata.TableName + `
+	stmt := `SELECT key, value, is_binary, etag, expiration_time FROM ` + a.metadata.TableName + `
 		WHERE
 			key = ?
 			AND (expiration_time IS NULL OR expiration_time > CURRENT_TIMESTAMP)`
 	ctx, cancel := context.WithTimeout(parentCtx, a.metadata.timeout)
 	defer cancel()
 	row := a.db.QueryRowContext(ctx, stmt, req.Key)
-	_, value, etag, err := readRow(row)
+	_, value, etag, expireTime, err := readRow(row)
 	if err != nil {
 		if errors.Is(err, sql.ErrNoRows) {
 			return &state.GetResponse{}, nil
 		}
 		return nil, err
 	}
 
+	var metadata map[string]string
+	if expireTime != nil {
+		metadata = map[string]string{
+			state.GetRespMetaKeyTTLExpireTime: strconv.FormatInt(expireTime.UnixMilli(), 10),
```

Comment:
I would prefer this be a ISO-8601 formatted date. UNIX timestamp milliseconds are less clear to handle. They also do not fit on a int32 which causes a number of problems with parsers.

Reply:
The problem with text based formats like ISO-8601 is that it is a bit slower than pure number based. We could use int32 unix time, but this would put a shelf life on Dapr to `2038`.

Reply:
it is a bit slower but i'm more concerned with correctness here. Also consistency: throughout Dapr we normally reference timestamps as ISO-8601 strings